python-sdk/nuscenes/tests/test_predict_helper.py

```diff
@@ -0,0 +1,475 @@
+import unittest
```

Comment:
Missing copyright blurb.

Reply:
Added the copyright blurb like the other nuscenes files.